extensions/interactions/MultipleChoiceInput/protractor.js

```diff
@@ -18,12 +18,16 @@
  */
```

Comment:
Is it possible to test this customization arg in the e2e test suite? I can't think of a straightforward way to do so. But if you have any ideas, please do share!

Reply:
Hmm, I tried to think of the possible test suites but the choices are randomly shuffled each time, I am also having the same doubt as you do.

Reply:
Maybe the test can be written to ensure that the following features work:
- Have say, 3 or 4 options and then enable this customization arg, and make sure that in the player view all the options are still present.
- Find the correct answer (maybe, loop through all the options), and then click it and ensure that Oppia says the answer is correct.
- Click a wrong answer and make sure Oppia reports it is wrong. 

I know you have tested this manually, but by adding it to the test suite, we have confirmation that no other code change will break this feature. Could you try to do this?